tests/testthat/test-util.R

```diff
@@ -161,6 +161,15 @@ test_that("git", {
   info <- git_info(path)
   expect_true(all(c("sha_short", "sha", "branch", "status") %in% names(info)))
 
+  nogit <- (Sys.getenv("NOT_CRAN") != "true")
```

Comment:
This should be done with withr, and always test against a valud path argument.

```
withr::with_options(
  list(orderly.nogit = TRUE),
  expect_null(git_info(path)))
```

you should do this in a new test block and start by unpacking the orderly git demo

Reply:
Done!